website/docs/concepts/modifiers/family.mdx

```diff
@@ -87,6 +87,9 @@ final characters = FutureProvider.autoDispose.family<List<Character>, String>((r
 });
 ```
 
+In this case however you can also achieve the same effect by using a single [StateNotifier] with a method that is called every time
```

Comment:
What is this supposed to be about?

Reply:
I just wanted to point out that using a family isn't always the best solution. The example I feel does not need a family, since multiple widgets wouldn't be listening to that provider with different arguments likely.

Reply:
I don't agree with this comment. This is most definitely a good use case for `family`.  
Nothing "need" `family` per say. It's supposed to simplify some use cases, which it does in this case.

And we can do debouncing with family+FutureProvider. StateNotifier is not necessary